addons/common/functions/fnc_deserializeObjects.sqf

```diff
@@ -9,6 +9,7 @@
  * 1: Center Position <ARRAY>
  * 2: Make Editable <BOOL> (default: true)
  * 3: Randomization <BOOL> (default: false)
+ * 4: Use Headless Client <BOOL> (default: true)
```

Comment:
This parameter doesn't exist so it always uses headless clients.

Reply:
6159bef
9987f62